app/views/govuk_publishing_components/components/_previous_and_next_navigation.html.erb

```diff
@@ -24,6 +28,14 @@
           data-track-label="<%= previous_page[:url] %>"
           data-track-dimension="previous"
           data-track-dimension-index="29"
+          <% if ga4_tracking %>
+            data-ga4-link = "<%= {
+              event_name: "navigation",
+              type: "content",
+              text: previous_page[:label],
```

Comment:
For both the next and previous labels, there's a check below to see if they exist. Can you write a quick test to check that this doesn't break if `ga4_tracking` is true but those labels don't exist?

Reply:
I added `|| title` in a future commit. Without the `|| title`, it was returning an empty string on the component renders that didn't have the label. Is this OK?